### ISSUE TITLE
Errors in CSV file downloaded in Advanced Search results

### ISSUE THREAD

**Issue by pmallol** [instedd/facility-public-portal#134]
Version: 3bbeb63

1. Navigate to Map
2. Expand Advanced Search view clicking the grey arrow next to Search bar
3. Search for facility name with letter 'G'
4. Click on 'Download results'

Obtained:
The search result shows 321 facilities found.
A CSV file is downloaded with only 5 results
The CSV file does not includes the services the facilities provide

```
"id","source_id","name","lat","lng","facility_type","ownership","contact_name","contact_email","contact_phone","location_1","location_2","location_3","location_4","services:en","services:am"
"62","1032973","Aba Gerga Health Post","7.2","36.9","Health Post","","","","","","","","","",""
"86","1031907","Abaya Gurucho Health Post","6.59","37.97","Health Post","","","","","","","","","",""
"108","1032001","Abela Gefeta Health Post","6.66","37.88","Health Post","","","","","","","","","",""
"122","1031746","Abera Galede Health Post","6.48","38.49","Health Post","","","","","","","","","",""
"129","1034738","Abeye Gurba Health Post","10.46","38.6","Health Post","","","","","","","","","",""
```
![screen shot 2016-12-13 at 4 27 11 pm](https://cloud.githubusercontent.com/assets/9214460/21155536/41a27ba0-c151-11e6-8bf8-a931ad0acbbc.png)

**Comment by pmallol** [comment]
Duplicate of #138

**Comment by juanedi** [comment]
Fixed csv download in c066d6b8e46d48a8609830cb508ca2f93431017f.

Note that in the examples the returned facilities don't have any known services. That's why they appear empty.
